src/bundle/Resources/views/content/content_edit/user_edit.html.twig

```diff
@@ -48,3 +48,18 @@
 {% block close_button %}
     <a class="ez-content-edit-container__close" href="{{ path('_ezpublishLocation', {'locationId': user.versionInfo.contentInfo.mainLocationId}) }}"></a>
 {% endblock %}
+
+{% block form_before %}
+    {{ ezplatform_admin_ui_component_group('content-create-form-before', {
```

Comment:
This is user edit shouldn't be `content-edit-form-before`?

Reply:
yes you are right, corrected and missing parameters added